### ISSUE TITLE
Ortographic profile

### ISSUE THREAD

**Issue by tresoldi** [intercontinental-dictionary-series/ids#8]
A profile for the entirety of IDS is complex, but some languages are low-hanging fruits, such as those with phonemic transcriptions (or phonemic-like, such as languages of Russia with a phonemic Cyrillic).

I'm starting working on it.

**Comment by LinguList** [comment]
I think it would be very helpful, @tresoldi, if we start with one
profile and one class of examples. I have another profile almost ready
for a subpart of IDS SEA languages, so if you look into a couple of
cyrillic orthographies now and feed back e.g., also with Ilya, so he can
have a quick look, I could later add the SEA profiles, and we'd be able
to do this step by step. But I think you'll need to adjust the code in
the IDS lexibank repo to leave languages unsegmented while segmenting
others. Note also that in many cases, words are placed in [] brackets
(at least in the SA part) to mark them as borrowings, which is a bit,
well, strange, but it means one has to be careful with the form spec,
and might even have to intervene here to extract borrowing information
with intention, or the like (maybe in a first pass).

**Comment by tresoldi** [comment]
Great! But depends on https://github.com/lexibank/pylexibank/issues/197

**Comment by LinguList** [comment]
Yes, I think, the alternative would be to pull out data and make a new
dataset, but I rather like to lift particular datasets a bit more, also
for ABVD, wehre I'd be keen on doing the SEA part, and one could add the
polynesian part by Mary, etc.
